sspl_test/functional_tests/v2/run_tests.sh

```diff
@@ -0,0 +1,382 @@
+#!/bin/bash
+
```

Comment:
pull files like this under sspl_test dir directly, which are not specific to functional v1 or v2 tests, its becoming otherwise too much duplicated code

put_config_to_consul.py
run_sspl-ll_tests.sh 
run_test.py 
run_tests.sh 
set_threshold.sh
sspl-test.spec
sspl_constants.py
sspl_start_checker